app/views/steps/shared/_approx_date_checkbox.html.erb

```diff
@@ -1,3 +1,3 @@
 <%= form.govuk_check_boxes_fieldset attribute, small: true, legend: { tag: 'span', size: 's', hidden: true } do %>
-  <%= form.govuk_check_box attribute, true, multiple: false, link_errors: true %>
+  <%= form.govuk_check_box attribute, true, false, multiple: false, link_errors: true %>
```

Comment:
This ensures forms with #govuk_check_boxes_fieldset are able to be submitted with a present but empty param instead of none. There is now a valueless #hidden_field added to the form to represent an unchecked state